packages/extension-vscode/CONTRIBUTING.md

```diff
@@ -19,10 +19,27 @@ for Visual Studio Code.
 ## Packaging
 
 * Install the packager via `npm install -g vsce`.
-* Run `npm install` on this folder (cannot package `yarn` install).
+* Run `npm install` from this folder (cannot package `yarn` install).
 * Run `vsce package`.
 * Install the generated `*.vsix` package in VSCode:
   * Go to `View > Extensions`.
   * Click `...` in the top-right of the panel.
   * Click `Install from VSIX...`.
   * Choose the generated `*.vsix` package from disk.
+
+## Publishing
+
+Full instructions available at the
+[Publishing Extensions](https://code.visualstudio.com/docs/extensions/publish-extension)
+page in the Visual Studio Code documentation. You must be a member of the
+[`webhint` Azure DevOps organization](https://webhint.visualstudio.com/)
+to publish.
+
+* Install the packager via `npm install -g vsce`
+* Run `npm install` from this folder (cannot publish `yarn` install).
+* Run `vsce login webhint`
+* Provide your [Personal Access Token](https://code.visualstudio.com/docs/extensions/publish-extension#_get-a-personal-access-token)
```

Comment:
Can you please convert all the links to link labels for consistency with the rest of the docs?